sdk/appconfiguration/app-configuration/samples/listRevisions.ts

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// This sample demonstrates how to list revisions for a configuration 
+// setting.
+
+// NOTE: replace with import { AppConfigurationClient } from "@azure/app-configuration"
+// in a standalone project
+import { AppConfigurationClient } from "../src"
+
+export async function run() {
+  console.log(`Running listRevisions sample`);
+
+  // You will need to set this environment variable
+  const connectionString = process.env["AZ_CONFIG_CONNECTION"]!;
+  const client = new AppConfigurationClient(connectionString);
+
+  // let's create the setting
+  const originalSetting = await client.addConfigurationSetting({
+    key: `keyWithRevisions-${Date.now()}`,
```

Comment:
Maybe don't have `date.now()` in the sample (IIUC it's mostly for testing purposes?)

Reply:
I normally avoid it as well for samples as well.

The only reason it's here is because settings, even when deleted, retain history for a set period of time. So I need to create a new setting in the sample each time so the list of revisions is more reasonable, rather than "from the beginning of the retention interval to now".